amplify-tools/amplify-tools-gradle-plugin/src/main/groovy/com/amplifyframework/tools/gradle/plugin/amplifytools.groovy

```diff
@@ -0,0 +1,246 @@
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.apache.tools.ant.taskdefs.condition.Os
+import groovy.json.JsonSlurper
+import groovy.json.JsonOutput
+
+class amplifytools implements Plugin<Project> {
```

Comment:
This is a weird name for a groovy class. Does it match some other token, somewhere else? Is this intentional? I would expect this to be in Pascal case, like `Plugin<Project>` is. `AmplifyToolsPlugin implements Plugin<Project>`, something like that?